lib/stripe.rb

```diff
@@ -119,6 +119,49 @@ def self.set_app_info(name, partner_id: nil, url: nil, version: nil)
       version: version,
     }
   end
+
+  class Preview
+    def self._get_default_opts(opts)
+      { stripe_version: ApiVersion::PREVIEW, api_mode: :preview }.merge(opts)
```

Comment:
We should send the `ApiVersion::PREVIEW` for all `api_mode: :preview` requests that go through `raw_request`

Reply:
We already do in https://github.com/stripe/stripe-ruby/pull/1217/files#diff-fe0e4dffe7e252a01f235338ad30c7e0343bd0369a074fe40e0330784349db0dR910